### PR TITLE
docs: update the doc of makefile target with lint_perltidy

### PR DESCRIPTION
**Pull request by albatrousse**
just casually adding a line with an explanation of the command `make lint_perltidy` and solution to a common issue with the latter command

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -35,4 +35,9 @@ See also [targets to run tests](../docs/dev/how-to-write-and-run-tests.md#runnin
 | `make clean`              | Clean up your dev environment: removes locally bound folders, run `hdown` and `prune`. | Run `make dev` to recreate a fresh dev env afterwards.        |
 | `make import_sample_data` | Load sample data (~100 products) into the MongoDB database.                            |                                                               |
 | `make import_prod_data`   | Load latest prod data (~2M products, 1.7GB) into the MongoDB database.                 | Takes up to 10m. Not recommended for dev setups !             |
+| `make lint_perltidy`      | Indent and reformat your code                         |
```
Comment (stephanegigandet) [inline docker/README.md:38]: maybe just "make lint" ? Easier to remember/type, and make lint is running make lint_perltidy

Comment (albatrousse) [reply docker/README.md:38]: Yeah, it is a good idea. I'll do this right now

File: Makefile [hunk 2]
```diff
@@ -334,7 +334,7 @@ check_perltidy:
 	test -z "${TO_TIDY_CHECK}" || ${DOCKER_COMPOSE} run --rm --no-deps backend perltidy --assert-tidy -opath=/tmp/ --standard-error-output ${TO_TIDY_CHECK}
 
 # same as check_perltidy, but this time applying changes
-lint_perltidy:
+lint:
```
Comment (alexgarel) [inline Makefile:337]: `lint` target already exists and depends `lint_perltidy` so you should not change this target name. The idea is that maybe at some point we want to add more lint commands.

Comment (albatrousse) [reply Makefile:337]: Ooh, I misunderstood Stephane 
`make lint` run directly all the "lint" commands

File: docker/README.md [hunk 3]
```diff
@@ -35,4 +35,10 @@ See also [targets to run tests](../docs/dev/how-to-write-and-run-tests.md#runnin
 | `make clean`              | Clean up your dev environment: removes locally bound folders, run `hdown` and `prune`. | Run `make dev` to recreate a fresh dev env afterwards.        |
 | `make import_sample_data` | Load sample data (~100 products) into the MongoDB database.                            |                                                               |
 | `make import_prod_data`   | Load latest prod data (~2M products, 1.7GB) into the MongoDB database.                 | Takes up to 10m. Not recommended for dev setups !             |
+| `make lint`      | Indent and reformat your code                         |
+
+PS : If you are having permission issues with `make lint` try writing the following commands :
```
Comment (alexgarel) [inline docker/README.md:40]: you can add footnotes, see https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#footnotes:


```suggestion
| `make lint`      | Indent and reformat your code[^lint]                       |

[^lint]: If you are having permission issues with `make lint` try writing the following commands :
```

**Review by alexgarel — APPROVED** [review]

**Pull request merged**